execution/executor-service/src/metrics.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_metrics_core::{
+    exponential_buckets, register_histogram, register_int_counter, Histogram, IntCounter,
+};
+use once_cell::sync::Lazy;
+
+pub static APTOS_REMOTE_EXECUTOR_CMD_RX_SECONDS: Lazy<Histogram> = Lazy::new(|| {
```

Comment:
`APTOS` prefix is redunant in the counter name - only relevant in the metrics name..